From 6715560de4d622c2d72ee7b587c916ac647c54bb Mon Sep 17 00:00:00 2001
From: Trevor Gamblin <tgamblin@baylibre.com>
Date: Fri, 6 Oct 2023 10:59:44 -0400
Subject: [PATCH] test_storlines: skip due to load variability

This is yet another test that intermittently fails on the Yocto AB when
a worker is under heavy load, so skip it during testing.

Upstream-Status: Inappropriate [OE-Specific]

[YOCTO #14933]

Signed-off-by: Trevor Gamblin <tgamblin@baylibre.com>
---
 Lib/test/test_ftplib.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Lib/test/test_ftplib.py b/Lib/test/test_ftplib.py
index 4c4a449..b8c79a4 100644
--- a/Lib/test/test_ftplib.py
+++ b/Lib/test/test_ftplib.py
@@ -629,6 +629,7 @@ class TestFTPClass(TestCase):
             self.client.storbinary('stor', f, rest=r)
             self.assertEqual(self.server.handler_instance.rest, str(r))
 
+    @unittest.skip('timing related test, dependent on load')
     def test_storlines(self):
         data = RETR_DATA.replace('\r\n', '\n').encode(self.client.encoding)
         f = io.BytesIO(data)
